src/Microsoft.Kusto.ServiceLayer/DataSource/Intellisense/KustoIntellisenseClient.cs

```diff
@@ -332,7 +326,7 @@ public LanguageServices.Contracts.CompletionItem[] GetAutoCompleteSuggestions(Sc
             foreach (var autoCompleteItem in completion.Items)
             {
                 var label = autoCompleteItem.DisplayText;
-                var insertText = autoCompleteItem.Kind == CompletionKind.Table
+                var insertText = autoCompleteItem.Kind == CompletionKind.Table || autoCompleteItem.Kind == CompletionKind.Database
```

Comment:
Added Database here to escape databases in autocomplete when they have special characters in the name.